.github/workflows/build-centos-7.yml

```diff
@@ -2,6 +2,10 @@ name: build-centos-7
 
 on:
   push:
+    paths-ignore:
+      # Don't run this workflow if only files under doc/ have been
+      # modified.
+      - "doc/**"
```

Comment:
Should be applied to the other workflows as well.

Reply:
with exception of doc builders :)